### DISCUSSION TITLE
Grid/tile view for artists

### DISCUSSION THREAD

**Discussion by nothing2obvi** [navidrome/navidrome#3065] [Ideas]
I saw a closed feature request about this idea so I opened this discussion.

It would be great if Navidrome had a grid view for artists. It's much more visually appealing to showcase the artist photos many of us have carefully curated. It's also usually how my mind works when I want to select music.

**Comment by deluan** [comment]
This has been proposed a few times, and will be implemented in the new UI being worked on.

**Comment by nothing2obvi** [comment]
That's great to hear! Thanks for replying.

**Comment by revenz** [comment]
Any update on this?    Didnt want to make yet another ticket for this, but would love to see a grid view for artists.
